core/regions/src/test/java/software/amazon/awssdk/regions/internal/util/EC2MetadataUtilsServer.java

```diff
@@ -110,6 +110,10 @@ private void handleConnection(BufferedReader input,
 
         } else if (path.equals("/latest/dynamic/instance-identity/document")) {
             outputInstanceInfo(output);
+
+        } else if (path.equals("instance-identity/signature")) {
```

Comment:
Hmm, integ tests failed 

```
RROR] software.amazon.awssdk.regions.util.EC2MetadataUtilsIntegrationTest.testInstanceSignature(software.amazon.awssdk.regions.util.EC2MetadataUtilsIntegrationTest) 
[ERROR]   Run 1: EC2MetadataUtilsIntegrationTest.testInstanceSignature:117 expected:<foobar> but was:<null> 
[ERROR]   Run 2: EC2MetadataUtilsIntegrationTest.testInstanceSignature:117 expected:<foobar> but was:<null> 
[ERROR]   Run 3: EC2MetadataUtilsIntegrationTest.testInstanceSignature:117 expected:<foobar> but was:<null> 
```

I think this should be ` else if (path.equals("/latest/dynamic/instance-identity/signature"))`

Reply:
Fixed